src/main/java/net/geforcemods/securitycraft/blocks/reinforced/ReinforcedSnowyDirtBlock.java

```diff
@@ -0,0 +1,185 @@
+package net.geforcemods.securitycraft.blocks.reinforced;
+
+import net.geforcemods.securitycraft.SCContent;
+import net.geforcemods.securitycraft.SCTags;
+import net.geforcemods.securitycraft.api.OwnableTileEntity;
+import net.geforcemods.securitycraft.misc.OwnershipEvent;
+import net.minecraft.block.*;
+import net.minecraft.block.material.Material;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.BlockItemUseContext;
+import net.minecraft.item.ItemStack;
+import net.minecraft.particles.ParticleTypes;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.Direction;
+import net.minecraft.util.NonNullList;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IBlockReader;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.World;
+import net.minecraft.world.gen.feature.ConfiguredFeature;
+import net.minecraft.world.gen.feature.DecoratedFeatureConfig;
+import net.minecraft.world.gen.feature.FlowersFeature;
+import net.minecraft.world.server.ServerWorld;
+import net.minecraft.world.storage.loot.LootContext;
+import net.minecraftforge.common.IPlantable;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.common.PlantType;
+
+import java.util.List;
+import java.util.Random;
+
+public class ReinforcedSnowyDirtBlock extends SnowyDirtBlock implements IReinforcedBlock, IGrowable
+{
+	private Block BLOCK;

```

Comment:
For your information: Only static final fields' identifiers should be all uppercase as per convention. You don't need to change this, as I will go through this file later on when I can find the time and clean it up, as most of this seems to be vanilla code.